packages/config/config/devices/templates/master_template.json

```diff
@@ -43,7 +43,7 @@
 		"valueSize": 1,
 		"minValue": 0,
 		"maxValue": 1,
-		"defaultValue": 1,
+		"defaultValue": 0,
```

Comment:
There was 2 uses of this template and they needed defaultValue = 1.

I had 14 uses of this template that needed defaultValue = 0.

To me it made more sense to change this template but please tell me if I'm wrong.

Reply:
You cannot change the master template. It's used in dozens of other places.

Reply:
As my comment said it is used in 2 places. Which I both checked locally and https://github.com/zwave-js/node-zwave-js/search?q=base_enable_disable_inverted . And I updated both those places and added defaultValue=1 in here https://github.com/zwave-js/node-zwave-js/pull/2051/files#diff-cfea19ef962377943d781f23350480971815226082ba0db86dcfbe10d9e1ab01